services/githubService.js

```diff
@@ -106,6 +106,25 @@ function getFetch(url) {
 function getFetchWithAuthToken(url, params = null, headers = null) {
   return utils.fetch(url, "get", params, null, headers);
 }
+
+/**
+ * Fetches the user github informations by userId using GitHub API
+ * @param githubUsername {string} - githubUsername String
+ */
+
+const fetchUserGithubInfo = async (githubUsername) => {
```

Comment:
can we rename it to fetchGithubInfo?

and since the function already says github info lets rename the parameter as username.